src/components/utility/LoadingButton/template.html

```diff
@@ -0,0 +1,4 @@
+<button class="loading-button" @click="handleClick" :disabled="loading">
+  <i class="fa fa-refresh fa-spin" aria-hidden="true" v-show="loading"></i>
+  <span class="loading-button__label"><slot>Submit</slot></span>
```

Comment:
__ is a notation from BEM, a css naming methodology. http://getbem.com/introduction/ , indicate that the <span> is a Element of loading-button Block.